testsuite/features/secondary/min_centos_monitoring.feature

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@centos_minion
+Feature: Monitor SUMA environment with Prometheus on a CentOS minion
+  In order to monitore SUSE Manager server
+  As an authorized user
+  I want to enable Prometheus exporters
+
+  Scenario: Pre-requisite: enable Prometheus exporters repository on the CentOS minion
+    And I enable repository "tools_pool_repo" on this "ceos_ssh_minion"
```

Comment:
Typo: 'cenos' -> 'centos'. 
Otherwise looks good to me!

Reply:
no, it's (ugly) conventional names